src/main/java/org/humancellatlas/ingest/core/web/Links.java

```diff
@@ -112,6 +112,10 @@ public class Links {
     public static final String SUBMISSION_DOCUMENTS_SM_URL = "/documentSmReport";
     public static final String SUBMISSION_DOCUMENTS_SM_REL = "documentSmReport";
 
+    // Links to entities for projects
+    public static final String AUDIT_LOGS_URL = "/audit-logs";
```

Comment:
Why the kebab case? It is inconsistent with the rest of the urls.

Reply:
agreed. I read up that kebab-case is preferred for links. But it makes sense to stick to the existing url format

Reply:
updated the url